### DISCUSSION TITLE
How to show all contributions on profile?

### DISCUSSION THREAD

**Discussion by Ehtesham25** [community/community#57400] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

I have a GitHub account it shows my "contributions in the last year". Which are 2k+ when I add manually my total contributions (Years 2020, 2021, 2022, 2023) it becomes 2.5k+. Now I want to show my all contributions instead of only the last 12 months contribution. How can I do?

**Comment by radar-blip** [comment]
I don't think you can do that, the chart is designed to show only a year at a time.

**Reply by Ehtesham25** [reply]
we should request to GitHub please show our all contributions. :)

**Reply by radar-blip** [reply]
You can request it. I'm not sure how though, other than posting it here.
